csharp/autobuilder/Semmle.Autobuild.Shared/Autobuilder.cs

```diff
@@ -370,6 +370,20 @@ void exitCallback(int ret, string msg, bool silent)
             }
         });
 
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
Why is this needed?

Reply:
This is suggested by [CA1816](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1816). This is implementing the dispose pattern in case of non-sealed classes.